Submission (testcase is incorrect?)

nhioe opened the pull request:
I think there's an issue with the test case?
Given that braile input in test case 3, output should be Abc 234, not Abc 123.
I believe this also affects the unit test written for this.

nhioe@uwaterloo.ca

doublederek commented:
Hi there, I've updated the test file to reflect the proper number output, please pull down the changes and make any updates your PR as needed. Leave a comment when you do and I'll run the test. Thanks!

nhioe commented:
> Hi there, I've updated the test file to reflect the proper number output, please pull down the changes and make any updates your PR as needed. Leave a comment when you do and I'll run the test. Thanks!

Fixed! Thank you!
